autoprotocol/instruction.py

```diff
@@ -250,7 +257,22 @@ def __init__(self, groups, magnetic_head):
                 f"container_types: {self.heads[magnetic_head]} for head_type: "
                 f"{magnetic_head}"
             )
-
+        non_valid_well_volumes = [
```

Comment:
If you want a bit of separation this init validation could be a classmethod that you call and has access to the class var `working_vols`